components/QrScanner.tsx

```diff
@@ -77,7 +90,7 @@ export const QrScanner: React.FC<QrScannerProps> = (props) => {
           {props.title}
         </Text>
       )}
-      <Column margin="24 0" crossAlign="center">
+      <Column margin="18 0" crossAlign="center">
```

Comment:
please extract the styles to theme files.

Reply:
Following the previous code, the props **margin** and **aligning** properties in the Row or Column are implemented like this.

Reply:
let us make some refactoring as and when we needed. since we are making some changes here, let us refactor this a bit.